### PR TITLE
Laney Moran

### PR DESCRIPTION
**Pull request by DelaneyMoran**
edited HW-0

### CODE REVIEW

File: HW-0.Rmd [hunk 1]
```diff
@@ -9,6 +9,6 @@ Your tasks!
 
 * Change the name of the author above to yours
 * Change the date
-* Replace the NHL URL with the URL for the document you just posted to Rpubs **here**: [My RPubs Document](https://www.nhl.com/)
+* Replace the NHL URL with the URL for the document you just posted to Rpubs **here**: [My RPubs Document](http://rpubs.com/DelaneyMoran/154869)
```
Comment (rudeboybert) [inline HW-0.Rmd:12]: Good job. Rpubs is a quick way to post R code/output to the web so you can share your work with collaborators.

**Comment by rudeboybert** [comment]
An hour is about the max I'd like a student spending on git related things. Just like you did yesterday, if you get stuck again, please see me. 

I'm going to "close and comment" this pull (i.e. merge) request so that I don't actually merge your changes with the master copy of the HW.

**Comment by DelaneyMoran** [comment]
Oh, that was because I included yesterday's office hours. I spent less than 5 min on it today!

**Comment by rudeboybert** [comment]
sounds good.

**Pull request closed without merging**